spy/vm/astframe.py

```diff
@@ -834,6 +849,28 @@ def eval_expr_NameOuterCell(self, name: ast.NameOuterCell) -> W_MetaArg:
         color: Color = "blue" if sym.varkind == "const" else "red"
         return W_MetaArg(self.vm, color, w_T, w_val, name.loc, sym=sym)
 
+    def eval_expr_AssignExpr(self, assignexpr: ast.AssignExpr) -> W_MetaArg:
+        specialized = self.specialized_assignexprs.get(assignexpr)
+        if specialized is None:
+            specialized = self._specialize_AssignExpr(assignexpr)
+            self.specialized_assignexprs[assignexpr] = specialized
+        if isinstance(specialized, ast.AssignLocal):
+            return self._execute_AssignLocal(assignexpr.target, assignexpr.value)
+        elif isinstance(specialized, ast.AssignCell):
+            return self._execute_AssignCell(
+                assignexpr.target, specialized.target_fqn, assignexpr.value
+            )
+        else:
+            assert False
```

Comment:
I'm a bit confused here: in `ast.py` you introduce `ast.AssignExprLocal` and `ast.AssignExprCell`, but then you never actually instantiate them?
I expected that `_specialized_AssignExpr` would return one of those, but in reality it returns an `ast.Assign`, which is a stmt instead of an expr.

Can you explain what's going on and that is the reasoning behind?